src/keycloak/keycloak_admin.py

```diff
@@ -3825,6 +3880,27 @@ def get_authenticator_config(self, config_id: str) -> dict:
         )
         return raise_error_from_response(data_raw, KeycloakGetError)
 
+    def create_execution_config(self, execution_id: str, payload: dict) -> bytes:
+        """
+        Add autenticatorConfig to the execution.
+
+        AuthenticatorConfigRepresentation
+        https://www.keycloak.org/docs-api/24.0.2/rest-api/index.html#_authenticatorconfigrepresentation
+
+        :param execution_id: id of execution
+        :type execution_id: str
+        :param payload: config to add to the execution
+        :type payload: dir
+        :return: Response(json)
+        :rtype: dict
+        """
+        params_path = {"id": execution_id, "realm-name": self.connection.realm_name}
+        data_raw = self.connection.raw_post(
+            urls_patterns.URL_ADMIN_FLOWS_EXECUTION.format(**params_path) + "/config",
```

Comment:
For consistency, this would be preferable to be defined as a constant in `urls_patterns`, so something like

```
URL_ADMIN_FLOWS_EXECUTION_CONFIG = f"{URL_ADMIN_FLOWS_EXECUTION}/config"
```